src/ansys_sphinx_theme/theme/ansys_sphinx_theme/components/search-button-field.html

```diff
@@ -0,0 +1,14 @@
+
+<div class="search-button" id="search-button">
+<form class="bd-search d-flex align-items-center" action="{{ pathto('search') }}" method="get">
+  <i class="fa-solid fa-magnifying-glass"></i>
+  <input type="search" class="form-control" name="q" placeholder="{{ theme_search_bar_text }}"
+         aria-label="{{ theme_search_bar_text }}" autocomplete="off" autocorrect="off"
+         autocapitalize="off" spellcheck="false"/>
+  <span class="search-button__kbd-shortcut">
+    <kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd>K</kbd>
+  </span>
+</form>
```

Comment:
Let's add a border to the form and drop the one for the icon and the input. This way, it gives the impression that everything is unified under the same object.

Reply:
updated